shared/common-adapters/box.native.js

```diff
@@ -64,8 +64,8 @@ class Box2 extends React.Component<Box2Props> {
     return box2(this.props)
   }
 }
-const VBoxGap = ({gap}) => <View style={{height: globalMargins[gap]}} />
-const HBoxGap = ({gap}) => <View style={{width: globalMargins[gap]}} />
+const VBoxGap = ({gap}) => <View style={{flexShrink: 0, height: globalMargins[gap]}} />
```

Comment:
don't allow these gaps to shrink. honestly i think its a better default for box2 in general but that should happen after the release (ticketed)